enhancements/kubelet/split-filesystem.md

```diff
@@ -0,0 +1,282 @@
+---
+title: split-filesystem
+authors:
+  - kannon92
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - TBD
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - TBD
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - TBD
+creation-date: 2024-07-30
+last-updated: 2024-07-30
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/OCPNODE-2461
+see-also:
+  - "https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4191-split-image-filesystem/README.md"
+---
+
+# Split Filesystem
+
+## Open Questions
+
+- Installer does not support creating openshift clusters with multiple disk.
+  Does this feature have value without users being able to configure their cluster to have a separate filesystem?
+
+- Do we need a drop in configuration for container storage?
+  - https://github.com/containers/storage/pull/1885
```

Comment:
I have this open for a discussion as I'm not sure if we want this functionality or not at the moment. It sounded like for dev preview we would maybe want a drop in config because having a customer overwrite the entire storage conf is way more error prone than changing a single field.

But for tech preview, we maybe don't need to go this route. I'll leave this as is so we can continue to discuss pros/cons.

Reply:
I think we decided to not go this route as opening up configuration for storage puts us on the hook to support a lot of edge cases.